### ISSUE TITLE
Java Client Import via Maven Dependency Broken

### ISSUE THREAD

**Issue by perlboy** [ConsumerDataStandardsAustralia/java-artefacts#3]
**Description**
Using a standard Maven configuration of:
```
<dependency>
    <groupId>au.org.consumerdatastandards</groupId>
    <artifactId>client</artifactId>
    <version>0.9.6-SNAPSHOT</version>
</dependency>

```

The `client` artefact is not found because the parent pom does not appear to be published:
```
[ERROR] Failed to execute goal on project cds-project: Could not resolve dependencies for project cds-project:jar:0.0.1-SNAPSHOT: Failed to collect dependencies at au.org.consumerdatastandards:client:jar:0.9.6-SNAPSHOT: Failed to read artifact descriptor for au.org.consumerdatastandards:client:jar:0.9.6-SNAPSHOT: Could not find artifact au.org.consumerdatastandards:cds-java-artefacts:pom:0.9.6-SNAPSHOT in central (https://repo.maven.apache.org/maven2)
```

Also fails on stable releases on at least `0.9.5`

**Related Work Items**
Where applicable please supply a list of related work items (using #issue format):
    - #1

**Comment by fyang1024** [comment]
Thank you @perlboy for raising the issue. 
I have fixed 0.9.6-SNAPSHOT releases. (The fix is in 0.9.6 branch and will be merged into master in a couple of days)
0.9.6 final release will be ready in a couple of days. 
Please let me know if the issue has been fixed.

**Comment by fyang1024** [comment]
I have tested 0.9.6 release and it seems working now, hence I would close this issue. Unfortunately we cannot go back to fix 0.9.3, 0.9.4 and 0.9.5 artefacts, but hopefully no one will really need to use them in future.
